src/main/java/seedu/address/model/user/UserList.java

```diff
@@ -0,0 +1,69 @@
+package seedu.address.model.user;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Iterator;
+import java.util.List;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+
+/**
+ * This class exists because I cannot find a way of making an unmodifiable/observable standalone value for Users.
+ */
```

Comment:
I just took another look at `ObservableValue`, turns out there are some subclasses we could potentially use.

[`ReadOnlyObjectWrapper`](https://docs.oracle.com/javase/8/javafx/api/javafx/beans/property/ReadOnlyObjectWrapper.html) can provide the observable wrapper for a `User`, and return an unmodifiable property to a getter with `ReadOnlyObjectWrapper::getReadOnlyProperty`.

I was also considering having multiple `User`s for different people using the same app, but that goes against the tP constraints so never mind.